README.md

```diff
@@ -144,6 +144,28 @@ vars:
 vars:
   hubspot__pass_through_all_columns: true # default is false
 ```
+
+### Adding property label
+For `property_hs_*` columns, you can enable the corresponding, human-readable `property_option`.`label` to be included in the staging models. 
+- **Note** you cannot bring in labels if using `hubspot__pass_through_all_columns: true`.` 
+- To enable labels for a given property, set the property attribute `use_property_label: true`, using the below format.
+
+```yml
+vars:
+  hubspot__ticket_pass_through_columns:
+    - name: "property_hs_fieldname"
+      alias: "fieldname"
+      use_property_label: true
```

Comment:
This is inconsistent with the code. In the code you have this as the `add_property_label` argument. Which should it be? I imagine it would be easiest to update the documentation to match the code.

Reply:
Please make the appropriate updates throughout the PR